### ISSUE TITLE
[Feature request] Play sound when file uploaded.

### ISSUE THREAD

**Issue by ReveltyOnGit** [timvisee/send#139]
Hello,
I'm looking for a way to add a sound when the transfer is complete, it would be nice to add this feature.
If someone has already done it, how?
Thanks in advance and good luck with this project!

**Comment by timvisee** [comment]
Thanks for the suggestion.

I am however not a fan of sudden sounds on the internet (except for videos). And I'm sure many agree. Besides, this requires extra permissions in many browsers.

Currently, the upload progression is already shown in the favicon and page title. Is that enough?

What do others think about a feature like this?

**Comment by BotMaster3000** [comment]
An alternative would be to allow Push-Notifications

https://support.mozilla.org/en-US/kb/push-notifications-firefox

**Comment by timvisee** [comment]
I agree that notifications may be a nice middle ground. It requires explicit permissions though.
